editor/src/components/canvas/canvas-strategies/strategies/flow-reorder-strategy.tsx

```diff
@@ -65,7 +65,7 @@ export function flowReorderStrategy(
       interactionSession != null &&
       interactionSession.interactionData.type === 'DRAG' &&
       interactionSession.activeControl.type === 'BOUNDING_AREA'
-        ? 3
+        ? 1
```

Comment:
Why has this fitness value changed?

Reply:
Flow reorder allows you to leave the parent, so this way reparent to flow would never win, even when you drag over a new parent.